introduction/modules/modules.md

```diff
@@ -7,10 +7,28 @@
 SmartDocuments itself is an external service that deals with generating documents based on templates and placeholders.
 This module offers a plugin which can connect to this service to generate documents.
 
+## OpenZaak
+
+[OpenZaak](https://openzaak.org/) is an application that implements APIs for the exchange of information concerning cases
+using the Dutch government API standards for ZGW (Zaakgericht Werken). This module provides a plugin that provides 
+authentication for other ZGW plugins.
+
 ## Plugins
 
 Plugins are extensions on the Valtimo platform. These are often connections to external
 services, such as transactional email and  generating documents. Some plugins are provided
 by the platform, but developers can create their own plugins as well. Plugins can be 
 configured without writing code, and can be used during BPMN processes. A more in-depth
 description can be found [here](plugin-introduction.md).
+
+## ZGW (Zaakgericht Werken)
+
+### Documenten API
+
+The Documenten (Documents) API is a Dutch government API standard meant to standardize the storage of and access to 
```

Comment:
A standard is already meant to standardize. Maybe rephrase this to:
The Documenten (Documents) API is a Dutch government API standard for storage of and access to...